src/WebJobs.Script.WebHost/Diagnostics/LinuxContainerEventGenerator.cs

```diff
@@ -100,6 +102,7 @@ private void ConsoleWriter(string evt)
 
         public override void LogAzureMonitorDiagnosticLogEvent(LogLevel level, string resourceId, string operationName, string category, string regionName, string properties)
         {
+            _writeEvent($"{ScriptConstants.LinuxAzureMonitoringEventStreamName} {(int)ToEventLevel(level)},{resourceId},{operationName},{category},{regionName},{properties}");
```

Comment:
How does turning this on/off work? 
We always write the events and and MA will use the latest config to decide if it needs to go any further?

Reply:
No. we decide which version of MA config to deploy. When we want to turn it on, we will move to the config version which has azure monitoring enabled

Reply:
I meant if the customer turns off log analytics for a particular site or it was never turned on do we still emit events (and write to file )?

Reply:
yes. but like Brett said, the MA agent won't upload it to Geneva.

Reply:
Got it. Concern is on Linux we use the temp files before emitting the eventual etw events. With a large number of containers writing these logs disk space could become a bottleneck + more stuff for the agent to process. 
If there was a way to know a particular site has this enabled then we can inject it to the container during specialization.

@brettsam is there an expectation that once this turned on by the customer for a site , logs that were emitted before it was enabled will also be available in the customer tables?

Reply:
No expectation that they exist before it was turned on. I know that there was discussion a while back about having Antares track whether this was enabled. I don't know if that ever happened, though. Ruslan may know -- I think it was his team.

Reply:
We don't need to worry about disk space yet. The functions log directory was moved to a bigger disk a while back.